tests/test_tradingcalendar.py

```diff
@@ -26,7 +26,6 @@
 
 class TestTradingCalendar(TestCase):
 
-    @nottest
```

Comment:
I can't remember the circumstances exactly, but IIRC we had to overhaul the calendar for some reason, and only had the time to port the U.S. Equity calendar.

(Hoping the more flexible calendar will help not having to make that tradeoff again.)